profiles/upload.yaml

```diff
@@ -74,6 +74,7 @@ recency:
 builds:
   h1n1pdm_2y:
     lineage: h1n1pdm
+    prioritized_seqs_file: "profiles/upload/h1n1pdm/{segment}/prioritized_seqs_file.tsv"
```

Comment:
I can see why these files live close to the `upload` profile, since that is where we are using them first, but I think they should probably live in the top-level `config/` directory (e.g., `config/h3n2/ha/prioritized_seqs_file.tsv`). This placement would make the files more discoverable alongside other lineage-specific configs and encourage their reuse in other builds. Even though we plan to get rid of fauna eventually, it's still possible we could want a profile with builds that use the download from fauna rule and these prioritized sequences.

Reply:
> This placement would make the files more discoverable alongside other lineage-specific configs and encourage their reuse in other builds

Hah, I put these in the `upload` profile for the exact opposite reason. I wanted to hide them from the general configs since they aren't used in the core workflow and might be confused as unused files.

Reply:
Moved to the top level config in d59e5bd